Homework/Maxim Semenko/lesson_37_Threading/les37_task1.py

```diff
@@ -0,0 +1,22 @@
+import threading
+
+
+class Counter(threading.Thread):
+    counter = 0
+    rounds = 100_000
```

Comment:
The task says: "Make the class have two global variables...". Your variables are not global.